a2lfile/src/a2ml.rs

```diff
@@ -118,13 +120,14 @@ pub enum GenericIfData {
 
 // tokenize()
 // Tokenize the text of the a2ml section
-fn tokenize_a2ml(input: &str) -> Result<Vec<TokenType>, String> {
+fn tokenize_a2ml(filename: Option<String>, input: &str, complete_string: &mut String) -> Result<Vec<TokenType>, String> {
```

Comment:
Is there a valid situation where the filename is None? It this is just about the testcases, then this should just be a String, and the testcases can use String::new() as their filename.

Reply:
done, i have left the check for the string being empty in order to be able to run the /include unit tests